lib/notifier/webhook.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios')
+const _ = require('lodash/fp')
+const uuid = require('uuid')
+
+const sendMessage = (settings, rec) => {
+  const webhookUrl = process.env.WEBHOOK_URL
+
+  if (_.isEmpty(webhookUrl)) return Promise.resolve()
```

Comment:
If the above is possible, then there's no need to penalize those who don't use webhooks:

```js
const sendMessage = _.isEmpty(webhookUrl) ?
  () => Promise.resolve() :
  (settings, rec) => /* ... */
```